src/components/error-overlay/error-overlay.tsx

```diff
@@ -123,8 +125,26 @@ class ErrorOverlay extends Component<any, any> {
    * @memberof ErrorOverlay
    */
   renderErrorHead(): VNode<any> | undefined {
-    const errorMessage = navigator.onLine ? 'error.default_error' : 'error.network_error';
-    return <div className={style.headline}>{this.props.errorHead ? this.props.errorHead : <Text id={errorMessage} />}</div>;
+    const {errorCategory, errorTitle, errorMessage} = this.props.errorDetails;
+    let errorTitleRes: any = '',
+      errorMessageRes: any = '';
+    if (errorTitle && errorMessage) {
```

Comment:
hmm so I asked Product, meanwhile we assume that every errorDetails that is passed from the outside (external plugins), should provide both of them. in theory we might get only one of them but it doesn't really make sense since all errors should have title and message. however, if it does happen, the logic will get it by category id and if the category is not core, then it will fallback to "Something went wrong" generic message.